### PR TITLE
Feat: Create contact from contacts page

### PR DESCRIPTION
**Pull request by nithindavid**
- Adds create contact form in a model

image

### CODE REVIEW

File: app/javascript/dashboard/routes/dashboard/conversation/contact/EditContact.vue [hunk 1]
```diff
@@ -11,6 +11,8 @@
         :contact="contact"
         :in-progress="uiFlags.isUpdating"
         :on-submit="onSubmit"
+        @success="onCancel"
```
Comment (pranavrajs) [inline app/javascript/dashboard/routes/dashboard/conversation/contact/EditContact.vue:14]: `onSuccess`

**Review by pranavrajs — CHANGES_REQUESTED** [review]
Please see the inline comments.

File: app/javascript/dashboard/routes/dashboard/conversation/contact/EditContact.vue (hunk 1 above)
Comment (nithindavid) [reply app/javascript/dashboard/routes/dashboard/conversation/contact/EditContact.vue:14]: I prefer not to use `on` before emitted events. as `@success` is already short for `v-on:success`. @pranavrajs What do you think ?

Comment (pranavrajs) [reply app/javascript/dashboard/routes/dashboard/conversation/contact/EditContact.vue:14]: I meant that the callback is wrong. `@success` is fine.

Comment (nithindavid) [reply app/javascript/dashboard/routes/dashboard/conversation/contact/EditContact.vue:14]: The idea was to call modal closing function after success, I'll create another function

Comment (nithindavid) [reply app/javascript/dashboard/routes/dashboard/conversation/contact/EditContact.vue]: Made the changes

**Review by pranavrajs — APPROVED** [review]

**Pull request merged**